mqlc/mqlc.go

```diff
@@ -1831,6 +1840,29 @@ func (c *compiler) updateEntrypoints(collectRefDatapoints bool) {
 		return res[i] < res[j]
 	})
 	c.block.Datapoints = append(c.block.Datapoints, res...)
+	// In the case of .all(...) queries, we have two datapoints for the list:
```

Comment:
I found this comment a little confusing. It only mentions all, however the implementation below applies to more cases than all.

Reply:
I tried to clarify the comment.

But I'm not sure whether I understood your comment entirely correctly. Is the comment confusing because we have the additional datapoint also for `.none(...)` and alike, or because the loop also runs for other functions (`.where(...)`, ...)? Should I also add a switch statement as I did for the other code changes in this PR?

Reply:
I think the changes you made are good. When I initially read it, it sounded like the logic should have only been applied for all.